presto-spi/src/main/java/com/facebook/presto/spi/type/UnscaledDecimal128Arithmetic.java

```diff
@@ -586,16 +588,12 @@ public static int compareUnsigned(Slice left, Slice right)
 
     public static int compareUnsigned(long leftRawLo, long leftRawHi, long rightRawLo, long rightRawHi)
```

Comment:
It actually does. It is only treated as a part of a number. This method works exactly as `Long.compareUnsigned`